nautobot/core/templates/500.html

```diff
@@ -4,13 +4,13 @@
 {% block title %}Server Error{% endblock %}
 
 {% block content %}
-<div class="row" style="margin-top: 150px;">
+<div class="row justify-content-center" style="margin-top: 9.375rem;">
     <div class="col-sm-4 col-sm-offset-4">
-        <div class="panel panel-default">
-            <div class="panel-heading">
-                <strong><i class="mdi mdi-alert"></i>Server Error</strong>
+        <div class="card">
+            <div class="card-header">
+                <strong class="hstack gap-4"><span class="mdi mdi-alert"></span>Server Error</strong>
             </div>
```

Comment:
As an aside - I've thought a few times about changing the default formatting of `panel-heading`/`card-header` to automatically bold (etc.) its contents, as I'm pretty sure we use the `<div class="card-header"><strong>...</strong></div>` nested HTML elements in most or all cases and it would be nice to remove that boilerplate.